src/utils/css.js

```diff
@@ -48,6 +48,12 @@ function classNames(prefix, modifierArgs) {
     return classes.join(' ')
 }
 
+export function componentStyles(styles) {
+    return function classNames(classy) {
+        return styles[classy]
```

Comment:
This is obviously the simplest possible function to return resolved values from the CSS modules, here we would support variable arguments, building up a complete class list from booleans, etc. which is needed runtime.